src/mumble/Messages.cpp

```diff
@@ -200,9 +230,9 @@ void MainWindow::msgServerSync(const MumbleProto::ServerSync &msg) {
 	{
 		// Since we are only loading the adjustments from the database, we don't really want to consider the adjustments
 		// to have "changed" by this action. Furthermore we are setting the volume adjustments before the listeners
-		// officially exist. Therefore some code that would receive the change-event would try to get the respective listener
-		// and fail due to it not existing yet.
-		// Therefore we block all signals while setting the volume adjustments.
+		// officially exist. Therefore some code that would receive the change-event would try to get the respective
+		// listener and fail due to it not existing yet. Therefore we block all signals while setting the volume
+		// adjustments.
```

Comment:
Unrelated change.

Reply:
That's the reason why I don't like running cling-format...

Reply:
The issue is actually that somebody forgot to run it...